### ISSUE TITLE
Incorrect error reported

### ISSUE THREAD

**Issue by yeskunall** [palantir/tslint#4260]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: Version 3.1.3
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```tsx
<div>This only works on mobile devices AFAIK. Try running this in a compatible browser on your mobile device.</div>
 <button onClick={vibrate(200)}>Vibrate for 200ms</button>
```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:recommended"
    ],
    "jsRules": {},
    "rules": {
      "quotemark": [true, "single", "jsx-double"]
    },
    "rulesDirectory": []
}

```

#### Actual behavior

<img width="1010" alt="screen shot 2018-11-01 at 8 34 11 am" src="https://user-images.githubusercontent.com/14703164/47852202-6815a900-ddb1-11e8-9f0e-98d188d890e8.png">

#### Expected behavior

The error should have been something like "React expects only one top-level div/container" as it does if I were to use ESLint.

**Comment by JoshuaKGoldberg** [comment]
@yeskunall I can't reproduce your error locally and suspect it's because I don't have the full file from the screenshot. Could you please post the file's contents, or generally some code that can showcase the issue on its own? Otherwise there's not much that can be done to help here.

Also, this looks suspicious:

```tsx
<button onClick={vibrate(200)}>Vibrate for 200ms</button>
```

Did you mean this?

```tsx
<button onClick={() => vibrate(200)}>Vibrate for 200ms</button>
```

**Comment by JoshuaKGoldberg** [comment]
Closing this for housekeeping - please do post here if you can provide more info @yeskunall!
